deploy/scripts/src/cri-lime/test.ts

```diff
@@ -228,61 +253,44 @@ export function fraud (): void {
         'SecondaryUKAddress.validUntilYear': ''
       },
       {
-        headers: { Authorization: `Basic ${encodedCredentials}` },
-        tags: { name: 'B01_Fraud_01_CoreStubEditUserContinue' }
+        headers: { Authorization: `Basic ${encodedCredentials}` }
       }
     ),
-    { isStatusCode200, ...pageContentCheck('We need to check your details') }))
+  { isStatusCode200, ...pageContentCheck('We need to check your details') }))
 
   sleepBetween(1, 3)
 
-  group('B01_Fraud_02_ContinueToCheckFraudDetails POST', () => {
-    res = timeRequest(() => res.submitForm({
-      params: {
-        redirects: 1,
-        tags: { name: 'B01_Fraud_02_ContinueToCheckFraudDetails_CRI' }
-      },
-      submitSelector: '#continue'
-    }),
-    { isStatusCode302 })
-    res = timeRequest(() => http.get(res.headers.Location,
-      {
-        headers: { Authorization: `Basic ${encodedCredentials}` },
-        tags: { name: 'B01_Fraud_02_ContinueToCheckFraudDetails_CoreStub' }
-      }
-    ),
-    { isStatusCode200, ...pageContentCheck('Verifiable Credentials') })
+  group(groups[1], () => { // B01_Fraud_02_ContinueToCheckFraudDetails
+    timeRequest(() => {
+      res = group(groups[2].split('::')[1], () => timeRequest(() => // 01_CRICall
+        res.submitForm({ params: { redirects: 1 }, submitSelector: '#continue' }), { isStatusCode302 }))
+      res = group(groups[3].split('::')[1], () => timeRequest(() => // 02_CoreStubCall
+        http.get(res.headers.Location, { headers: { Authorization: `Basic ${encodedCredentials}` } }),
+      { isStatusCode200, ...pageContentCheck('Verifiable Credentials') }))
+    }, {})
   })
   iterationsCompleted.add(1)
 }
 
 export function drivingLicence (): void {
-  let res: Response
   type drivingLicenceIssuer = 'DVA' | 'DVLA'
   const licenceIssuer: drivingLicenceIssuer = (Math.random() <= 0.5) ? 'DVA' : 'DVLA'
+  let groups: string[]
+  licenceIssuer === 'DVA' ? groups = groupMap.drivingLicence.filter((issuer) => issuer.includes('DVA')) : groups = groupMap.drivingLicence.filter((issuer) => issuer.includes('DVLA'))
```

Comment:
```suggestion
  let groups = groupMap.drivingLicence.filter(s => s.includes(licenceIssuer))
```
This can be simplified

Reply:
Thanks, this has been updated.